### PR TITLE
Added Traditional Chinese

### PR DESCRIPTION
**Pull request by plum7x**

### CODE REVIEW

**Review by nhirokinet — COMMENTED** [review]
Thank you for translation, can you check these comments?

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    <string name="preferences_category_urls">網頁與搜尋</string>
+    <string name="preferences_category_custom_commands">自訂命令</string>
+    <string name="preferences_category_home_screen">主畫面</string>
+    <string name="preferences_category_contacts">通訊錄</string>
+    <string name="preferences_category_advanced">進階</string>
+    <string name="preferences_category_about_this_app">關於</string>
+
+    <!-- Preferences items, main screen -->
+    <string name="preferences_item_main_show_startup_help_title">在啟動時顯示說明</string>
+    <string name="preferences_item_main_show_startup_help_summary">此選項只是向您顯示 help 指令的指南。在您記住 "help" 命令後，建議禁用此選項。</string>
```
[the remark this answers is not in the document]
Comment (nhirokinet) [reply app/src/main/res/values-zh-rTW/strings.xml:83]: Sorry this looks like from "help command" in original English version... looks like not a problem of translation

**Comment by plum7x** [comment]
I updated the translation to remove the superfluous words and correct the typos, it looks more comfortable now.

**Comment by nhirokinet** [comment]
Uploaded as version 1.2.18. Availability on Google Play and F-droid will take a time depending on each store's policy.

**Pull request merged**